Backport-Guide.md

```diff
@@ -77,19 +77,41 @@ After the backporting:
 ```
           +-----E-------G (stable-1.2 branch)
          /      ^       ^
-A---B---C---D---E---F---G (master branch)
+A---B---C---D---E---F---G (main branch)
 ```
 
 After the backport, the `stable-1.2` branch contains commits  `A`, `B` and
 `C`, `E` and `G`.
 
 ## Backport Workflow
 
+### Auto-backporting
+Auto-backporting is a helpful feature than can be used to speed up the backport process. The project uses [Backport Action](https://github.com/marketplace/actions/backport-action) to handle the backporting and another GitHub workflow to automatically add the `auto-backport` label.
+
+The following PRs will automatically have the `auto-backport` label added:
+- Corresponding issue contains the `bug` label
+- PR contains `backport-needed` label
+
+PRs with the `no-backport-needed` label will automatically have its `auto-backport` label removed, superseding the above conditions.
+
+Additionally, the `auto-backport` label can be added or removed manually.
+The check for labels is triggered whenever a label is added or removed from the PR.
```

Comment:
You could make this line a "[note](https://github.com/kata-containers/kata-containers/blob/main/docs/Documentation-Requirements.md#notes)]":

```suggestion
> **Note:** The check for labels is triggered whenever a label is added or removed from the PR.
```